dump-scripts/python/zend_to_horizen.py

```diff
@@ -25,54 +44,93 @@
 def satoshi_2_wei(value_in_satoshi):
 	return int(round(SATOSHI_TO_WEI_MULTIPLIER * value_in_satoshi))
 
-if len(sys.argv) != 3:
+if len(sys.argv) != 3 and len(sys.argv) != 5:
 	print(
-		"Usage: python3 {} <zend dump file name> <output_file>"
+		"Usage: python3 {} <zend dump file name> <mapping file name> <zend_vault_output_file> <eon_vault_output_file>"
```

Comment:
I'm worried that the possibility to specify manually the mapping file is more error-prone: I would prefer to have the mapping file  commited in this same repository (for legal reasons will also be commited directly by the exchange probably)  and  hardcode the import  directly in the python script. (probably we need 1 mapping per chain type (mainnet, testnet) , so we need to add the chain type as script argument)
what do you think?

Reply:
I see your point, but I wonder if the same logic applies also to the other files, like the zend dump, the eon dump etc.

Reply:
@cronicc what's your opinion on this?

Reply:
update: we will add an upper bash script in a separate pr that calls all the steps